### PR TITLE
SL supports sso with cloud cli

### PR DESCRIPTION
**Pull request by mirnawong1**
SL now supports SSO with the cloud cli. this pr updates the docs and also tweaks the exports sidebar label

### CODE REVIEW

File: website/snippets/_v2-sl-prerequisites.md [hunk 1]
```diff
@@ -7,4 +7,4 @@
 - Set up the [Semantic Layer API](/docs/dbt-cloud-apis/sl-api-overview) in the integrated tool to import metric definitions. 
   - dbt Core or Developer accounts can define metrics but won't be able to dynamically query them.<br />
 - Understand [MetricFlow's](/docs/build/about-metricflow) key concepts, which powers the latest dbt Semantic Layer.  
-- Note that SSH tunneling for [Postgres and Redshift](/docs/cloud/connect-data-platform/connect-redshift-postgresql-alloydb) connections, [PrivateLink](/docs/cloud/secure/about-privatelink), and [Single sign-on (SSO)](/docs/cloud/manage-access/sso-overview) doesn't support the dbt Semantic Layer yet.
+- Note that SSH tunneling for [Postgres and Redshift](/docs/cloud/connect-data-platform/connect-redshift-postgresql-alloydb) connections, and [PrivateLink](/docs/cloud/secure/about-privatelink) doesn't support the dbt Semantic Layer yet.
```
Comment (aiguofer) [inline website/snippets/_v2-sl-prerequisites.md]: I feel like this should be:

> Note that the dbt Semantic Layer doesn't yet support SSH tunneling for [Postgres and Redshift](/docs/cloud/connect-data-platform/connect-redshift-postgresql-alloydb) connections, and [PrivateLink](/docs/cloud/secure/about-privatelink).

Comment (mirnawong1) [inline website/snippets/_v2-sl-prerequisites.md]:
```suggestion
- Note that the dbt Semantic Layer doesn't yet support SSH tunneling for [Postgres and Redshift](/docs/cloud/connect-data-platform/connect-redshift-postgresql-alloydb) connections, and [PrivateLink](/docs/cloud/secure/about-privatelink).
```

Comment (mirnawong1) [reply website/snippets/_v2-sl-prerequisites.md]: great feedback, thank you @aiguofer ! folded it in now 🙏

**Review by aiguofer — APPROVED** [review]

**Review by rpourzand — COMMENTED** [review]
@mirnawong1  this is a bit more nuanced than just removing the below: 

* We support SSO only for the developer user in the Cloud CLI. 
* We don't support SSO for the Semantic Layer service account credential

**Comment by mirnawong1** [comment]
Oh ok great context @rpourzand ! I'll see to make it clearer 🙏

**Pull request merged**